packages/headless/src/api/commerce/search/response.ts

```diff
@@ -1,6 +1,12 @@
-import {Correction} from '../../search/search/query-corrections.js';
+import {QueryCorrection} from '../../search/search/query-corrections.js';
 import {CommerceSuccessResponse} from '../common/response.js';
 
 export interface SearchCommerceSuccessResponse extends CommerceSuccessResponse {
-  queryCorrection?: Correction;
+  queryCorrection?: SearchCommerceQueryCorrectionResponse;
+}
+
+interface SearchCommerceQueryCorrectionResponse {
```

Comment:
@Spuffynism I'm using this interface instead of `Correction` because while the CAPI Swagger indicates that `correctedQuery` and `originalQuery` should always return strings, I noticed that in practice they are actually nullable.

I think we should update the Swagger to reflect the actual behavior.